canonical-web-and-design/cn.ubuntu.com
# 15.10 cloud updates

## Done

15.10 updates for /cloud
## QA

Check against copydoc:
https://docs.google.com/document/d/1DoALrfAB1nE4I26fiJs1IYUVWvEAR6d0CVg5LMmlp8w/edit#heading=h.eaclxneatnj
## Card

https://canonical.leankit.com/Boards/View/111185042/117716116
